### PR TITLE
add `hexo-generator-readtime` plugin

### PR DESCRIPTION
**Pull request by lorezyra**
https://hexo.io/docs/plugins#Publishing

https://github.com/AsemAlhaidary/hexo-generator-readtime/issues/2

## Check List

**Please read and check followings before submitting a PR.**

- [ ] I want to publish my theme on Hexo official website.
  - [ ] I have read the [theme publishing doc](https://hexo.io/docs/themes#Publishing).
  - [ ] `name` is unique.
  - [ ] `link` URL is correct.
  - [ ] `preview` URL is correct.
  - [ ] `preview` URL web site is rendered correctly.
  - [ ] Add a screenshot to `source/themes/screenshots`.
  - [ ] Screenshot filename is same as value of `name`.
  - [ ] Screenshot size is `800 * 500`.
  - [ ] Screenshot file format is `png`.
- [x] I want to publish my plugin on Hexo official website.
  - [x] I have read the [plugin publishing doc](https://hexo.io/docs/plugins#Publishing).
  - [x] `name` is unique.
  - [x] `link` URL is correct.
- [ ] Others (Update, fix, translation, etc...)
  - Languages:
  - [ ] `en` English
  - [ ] `ru` Russian
  - [ ] `zh-cn` simplified Chinese
  - [ ] `zh-tw` traditional Chinese

> **Note**
>
> No need to translate `ko`, `pt-br`, `th` languages if you submit a PR for documentation.

> **Warning**
>
> `ko`, `pt-br`, `th` languages document are inactive. Please, submit issue if you want to continuous contribution to them.

### CODE REVIEW

**Comment by lorezyra** [comment]
Only need to add the readTime plugin (from the data file). Don't merge the other changes.

File: source/_data/plugins.yml [hunk 1]
```diff
@@ -3363,4 +3363,36 @@
     - tag
     - collapse
     - hide
-    - expand
\ No newline at end of file
+    - expand
+- name: hexo-generator-readtime
+  description: HEXO package that provides analytics on the read time to review a post. Generates word, character, image, and video counts in the front-matter of the markdown file. Plus provides read-time estimates for given language profile. Supports 42 languages!
```
Comment (MBRjun) [inline source/_data/plugins.yml:3368]: ``description: HEXO...``  
``HEXO`` -> ``Hexo``

**Review by MBRjun — CHANGES_REQUESTED** [review]
Also: Don't contain other changes, make another commit to restore ``.gitignore`` and ``.husky/pre-commit``

**Comment by lorezyra** [comment]
Apologies. Files restored.

**Review by yoshinorin — APPROVED** [review]
Thank you :)

**Pull request merged**